.github/renovate.json

```diff
@@ -1,6 +1,7 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
+    "config:recommended",
```

Comment:
Need this https://github.com/kachick/containers/commit/5923d37cdf8c0412ee5020107349fe22c011679b